core/src/main/java/ai/timefold/solver/core/config/localsearch/LocalSearchPhaseConfig.java

```diff
@@ -27,14 +27,17 @@
 import ai.timefold.solver.core.config.localsearch.decider.acceptor.LocalSearchAcceptorConfig;
 import ai.timefold.solver.core.config.localsearch.decider.forager.LocalSearchForagerConfig;
 import ai.timefold.solver.core.config.phase.PhaseConfig;
+import ai.timefold.solver.core.config.solver.PreviewFeature;
 import ai.timefold.solver.core.config.util.ConfigUtils;
+import ai.timefold.solver.core.impl.move.streams.maybeapi.stream.MoveProviders;
 
 import org.jspecify.annotations.NonNull;
 import org.jspecify.annotations.Nullable;
 
 @XmlType(propOrder = {
         "localSearchType",
         "moveSelectorConfig",
+        "moveProvidersClass",
```

Comment:
Should it be named `moveProviderClass`?

Reply:
Since the class is actually called "MoveProviders", I think this is a correct field name.
But I have the name of the class, so the field name will eventually change as well.